install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -749,6 +749,10 @@ data:
   vlan-bpf-bypass: {{ .Values.bpf.vlanBypass | join " " | quote }}
 {{- end }}
 
+{{- if hasKey .Values "netfilterCompatMode" }}
+  netfilter-compatible-mode: {{ .Values.netfilterCompatMode | quote }}
```

Comment:
Please add `netfilterCompatMode` to `install/kubernetes/cilium/values.yaml` with a top comment starting with `--` (see the other values). Then run  `make -C install/kubernetes docs` in order to update `install/kubernetes/cilium/README.md`.